### PR TITLE
Fix build with clang-10

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
**BEFORE**
On Ubuntu 20 with Clang 10, build doesn't work https://gist.github.com/nikitamikhaylov/18e5f22e81a89f701520558a4c39f30d
Because every statement is separated by ; and ; after { means empty statement, which has no effect.

**AFTER**
Build seems to work well.

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
Hi, @nikitamikhaylov . Thank you for your first contribution.
Does this PR include all fixes for clang-10?

**Comment by nikitamikhaylov** [comment]
> Hi, @nikitamikhaylov . Thank you for your first contribution.
> 
> Does this PR include all fixes for clang-10?

After that fixes I've successfully compiled it with clang-10 without any warnings. I hope that is all you need. 
You can try to include clang-10 in your CI :)

File: c++/test/TestBloomFilter.cc [hunk 1]
```diff
@@ -152,9 +152,9 @@ namespace orc {
 
     // test strings
     bloomFilter.reset();
-    const char * emptyStr = u8"";
-    const char * enStr = u8"english";
-    const char * cnStr = u8"中国字";
+    const char * emptyStr = "";
+    const char * enStr = "english";
+    const char * cnStr = "中国字";
```
Comment (dongjoon-hyun) [inline c++/test/TestBloomFilter.cc:157]: Did you run all tests too?

Comment (nikitamikhaylov) [reply c++/test/TestBloomFilter.cc:157]: http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0482r3.html
https://docs.microsoft.com/en-us/cpp/cpp/string-and-character-literals-cpp?view=vs-2019

There was funny warning from clang `type of UTF-8 string literal will change from array of const char to array of const char8_t in C++20`

And your CI said that everything is Ok.

Comment (dongjoon-hyun) [reply c++/test/TestBloomFilter.cc:157]: As you pointed out, CI doesn't run clang-10.
> You have clang-6 in your CI, but that is too old.

**Comment by dongjoon-hyun** [comment]
@nikitamikhaylov . Apache ORC has a dockerized testing environment like the following. When I run this this PR on clang9 on Ubuntu 20.04, this didn't pass the test. So, I'm wondering if this is only working on clang10.

- https://github.com/apache/orc/commit/0f33183e116789e6b37e38039d2746f6b59a4f0e

**Comment by dongjoon-hyun** [comment]
For this PR, ORC-647 added macOS 10.15 test to Travis CI with `clang12`. Could you rebase this PR to the master?

```
$ clang --version
90Apple clang version 12.0.0 (clang-1200.0.22.8)
91Target: x86_64-apple-darwin19.5.0
```

**Comment by dongjoon-hyun** [comment]
I closed this PR because there is no evidence that build is broken in clang-10. Please free free to reopen this with the evidence.

**Comment by nikitamikhaylov** [comment]
> I closed this PR because there is no evidence that build is broken in clang-10. Please free free to reopen this with the evidence.

@dongjoon-hyun Here you are: https://gist.github.com/nikitamikhaylov/18e5f22e81a89f701520558a4c39f30d

Do you know C++? :) Every statement is separated by `;` and `;` after `{` means empty statement, which has no effect.

**Comment by dongjoon-hyun** [comment]
It's unnecessarily insulting to me.
> Do you know C++?

As you said, the followings are just esthetic changes.
>  Every statement is separated by ; and ; after { means empty statement, which has no effect.

Could you show me your command line log? What was your failure in your environment?
> P.S. I am using Ubuntu 20 with Clang 10

I closed this PR because I want to reply from you. You didn't reply to the question for a long time.

**Comment by dongjoon-hyun** [comment]
Please include your comment on the PR description as `**BEFORE**` section.
- https://github.com/apache/orc/pull/517#issuecomment-668709736

Also, please include the log after this PR on the PR description as `**AFTER**` section.

Thanks for the reply.

**Comment by nikitamikhaylov** [comment]
I'm sorry, I didn't mean to offend you with a comment about C++. 😔

For some strange reason I had no notification for comments from this PR. Will do everything tomorrow.

**Comment by nikitamikhaylov** [comment]
:(

**Pull request closed without merging**